mono/mini/debugger-engine.h

```diff
@@ -154,6 +157,21 @@ typedef struct {
 	guint32 native_offset;
 } DbgEngineStackFrame;
 
+MonoGHashTable *
+mono_debugger_get_thread_states (void);
+
+gboolean
+mono_debugger_is_disconnected (void);
+
+intptr_t 
+mono_debugger_tls_thread_id (gpointer debuggerTlsData);
```

Comment:
Coding convention: variable name is `debugger_tls_data` and return value is `gsize`.